mon-pix/app/models/assessment.js

```diff
@@ -48,4 +48,9 @@ export default class Assessment extends Model {
     return answers.slice(sliceAnswersFrom);
   }
 
+  @computed('answers.[]')
```

Comment:
**suggestion (si mineure):** Enlever les computed de ce fichier

Reply:
J'ai retirer les computed, mais pas encore fait toutes les modifs pour pouvoir retirer l'exception sur `ember/no-computed-properties-in-native-classes`